packages/dashboard-core-plugins/src/panels/WidgetPanel.tsx

```diff
@@ -3,44 +3,47 @@ import classNames from 'classnames';
 import memoize from 'memoize-one';
 import { PanelComponent } from '@deephaven/dashboard';
 import type { Container, EventEmitter } from '@deephaven/golden-layout';
-import { ContextActions } from '@deephaven/components';
+import { ContextActions, createXComponent } from '@deephaven/components';
 import { copyToClipboard } from '@deephaven/utils';
 import Panel from './Panel';
 import WidgetPanelTooltip from './WidgetPanelTooltip';
 import './WidgetPanel.scss';
+import {
+  getWidgetPanelDescriptorFromProps,
+  WidgetPanelDescriptor,
+  WidgetPanelTooltipProps,
+} from './WidgetPanelTypes';
 
-interface WidgetPanelProps {
+type WidgetPanelProps = WidgetPanelTooltipProps & {
```

Comment:
Why does this extend the tooltip props? I thought the descriptor was passed to the panel